src/DotNetWorker/GrpcWorker.cs

```diff
@@ -129,18 +129,24 @@ internal async Task InvocationRequestHandlerAsync(StreamingMessage request)
             invocationFeatures.Set<FunctionInvocation>(invocation);
 
             FunctionContext context = _application.CreateContext(invocationFeatures);
+            try
+            {
+                context.Features.Set<IFunctionBindingsFeature>(new GrpcFunctionBindingsFeature(context, request.InvocationRequest, _outputBindingsInfoProvider));
```

Comment:
Setting on the context for safety (this could become a problem if the reference in the context wasn't the same passed in, which is possible)